src/components/Search/Search.js

```diff
@@ -117,8 +117,7 @@ class Search extends React.PureComponent {
   };
 
   handleFilterChange = (ev, value) => {
-    this.input.focus();
```

Comment:
Maybe we can keep this one as I think we'd want it in all cases and might be hard to do it externally without the `input` DOM reference.

Reply:
We can try that out.